Dockerfile

```diff
@@ -9,11 +9,14 @@ ENV LANG=C.UTF-8 \
     PYTHONDONTWRITEBYTECODE=1
 WORKDIR /kubespray
 COPY *yml .
+COPY *cfg .
```

Comment:
HI @dabeck , 
copy can only be worked with '/' when run with `docker build`
```
When using COPY with more than one source file, the destination must be a directory and end with a /
```

Reply:
Works on my maschine but I am willing to change this.